R/pkg/inst/tests/testthat/test_utils.R

```diff
@@ -17,6 +17,9 @@
 
 context("functions in utils.R")
 
+# Ensure Spark is installed
+sparkCheckInstall()
```

Comment:
Ah thats a great idea - Can you see if that works (unfortunately it needs manual verification) ?